gitpod-io/gitpod
# [gp cli] Tasks cmd

## Description
Adds the following commands to gitpod-cli:

```shell
$ gp tasks list
$ gp tasks attach <terminalAlias>
```

## Related Issue(s)
Fixes #7016

## How to test

1. Start gitpod in gitpod
2. `cd components/gitpod-cli`
3. `go get`
4. `go build main.go`
5. `./main tasks list`
6. `./main tasks attach 1`
7. kill the 1st task and see the new status appear

## Release Notes

```release-note
NONE
```

## Documentation
